include/tlapack/blas/lartg.hpp

```diff
@@ -0,0 +1,47 @@
+/// @file lartg.hpp
```

Comment:
Is it possible? Of course it is, but lartg genuinely makes a routine like svd_qr easier because it allows stuff like:
`lartg(d[i] * cs, e[i], cs, sn, r);`
whereas with rotg, this would be
```
T temp = d[i] * cs;
T temp2 = e[i]
rotg(temp, temp2, cs, sn);
```

Additionally, when translating unfamiliar routines (that likely already use lartg) it is just a little bit easier to use lartg.

Reply:
Hi. I see your point. So, I am fine with having `lartg`. Thanks for explaining!